src/main/java/org/folio/innreach/domain/service/impl/RequestServiceImpl.java

```diff
@@ -311,6 +313,9 @@ private void updateTransaction(InnReachTransaction transaction, InventoryItemDTO
     hold.setFolioItemId(item.getId());
     hold.setFolioItemBarcode(item.getBarcode());
     if (holding != null) {
+      var instance = instanceService.find(holding.getInstanceId()).orElse(null);
+      var author = instanceService.getAuthor(instance);
+      hold.setAuthor(author);
```

Comment:
author field should be set on our side only for item hold transactions, but method updateTransaction is applied for all types